### PR TITLE
NOISSUE - Add vendor and Makefile. Fix docker

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by drasko — CHANGES_REQUESTED** [review]
This is all noce, but can you please add LICENSE file, it is important.

**Review by drasko — APPROVED** [review]
LGTM

**Review by drasko — CHANGES_REQUESTED** [review]
Just one question - should we change repo name to `mfxkit`, or should we keep it as `service-template`?

@manuio @darkodraskovic opinions please.

**Comment by manuio** [comment]
> Just one question - should we change repo name to `mfxkit`, or should we keep it as `service-template`?
> 
> @manuio @darkodraskovic opinions please.

@drasko We can keep it and rename `mainflux/mfxkit` into `service-template/mfxkit`

**Comment by drasko** [comment]
This is exactly what I am asking - do we keep it as `service-template`, or we rename it to `mfxkit`?

**Comment by manuio** [comment]
> This is exactly what I am asking - do we keep it as `service-template`, or we rename it to `mfxkit`?

@drasko If we follow Mainflux repo logic this should be called `service-template/mfxkit`

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -8,18 +8,23 @@
 
 version: "3.7"
 
+networks:
+  docker_mainflux-base-net:
+    external: true
+
 services:
   mfxkit:
+    image: mainflux/mfxkit:latest
```
Comment (nmarcetic) [inline docker/docker-compose.yml:17]: Image still calls `mfkit` ? Not really sure if we need compose placeholder  (which can't work without the rest of Mainflux) ?

Comment (manuio) [reply docker/docker-compose.yml:17]: What do you mean? This works with the rest of Mainflux

Comment (nmarcetic) [reply docker/docker-compose.yml:17]: Yea, I was thinking do we need a docker-compose in the service template? 
We use a single docker-compose on the Mainflux level, not per service. So this composition can't work independently right (e.g this new service that you make from this template, needs to be added in `mainflux/docker/docker-compose`)? If so, this file is not usable.

Comment (manuio) [reply docker/docker-compose.yml:17]: I think so. It's useful to build and run the system fast without having to enter in "native" issues. We do use docker-compose per service for addons. This is a sort of external addon.

Comment (nmarcetic) [reply docker/docker-compose.yml:17]: Ok. Resolved.

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**